sys/linux/sys.txt

```diff
@@ -1032,7 +1033,7 @@ timer_flags = TIMER_ABSTIME
 utimensat_flags = AT_SYMLINK_NOFOLLOW
 priority_which = PRIO_PROCESS, PRIO_PGRP, PRIO_USER
 mempolicy_flags = MPOL_F_MEMS_ALLOWED, MPOL_F_ADDR, MPOL_F_NODE
-ptrace_req = PTRACE_LISTEN, PTRACE_KILL, PTRACE_INTERRUPT, PTRACE_ATTACH, PTRACE_DETACH
+ptrace_req = PTRACE_LISTEN, PTRACE_KILL, PTRACE_INTERRUPT, PTRACE_ATTACH, PTRACE_DETACH, PTRACE_TRACEME
```

Comment:
PTRACE_TRACEME was specifically blacklisted because it caused some issues in the past:
https://github.com/google/syzkaller/blob/master/sys/linux/init.go#L179-L181
I don't know if it's still relevant or not, but it's something that needs to be considered.

Reply:
OK, this explain why I couldn't reach such code. :)
I'll remove that for now.